executive-2023-11-29 (#2516)
makerdao/community: governance/votes/Executive vote - November 29, 2023.md

---

> title: Template - [Executive Vote] PSM-GUSD-A Removal from Cash Stablecoins, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vest Streams Updates and Spark Proxy-Spell - November 29, 2023

Reviewer: See our comment below re: cash stablecoins.

Suggested changed wording for vesting portion of title

```suggestion
title: Template - [Executive Vote] PSM-GUSD-A Debt Ceiling Reduction, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vesting Stream Updates and Spark Proxy-Spell - November 29, 2023
```

> summary: Setting PSM-GUSD-A Debt Ceiling to zero and removing from Autoline, Coinbase Custody (RWA014-A) Debt Ceiling increase, WBTC-A, WBTC-B, and WBTC-C Stability Fee changes, SBE parameter changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Cease Vest Streams for Steakhouse, Set up Ecosystem Facilitator Vest Streams and TechOps and Spark proxy-spell

Reviewer: See comment lower down. The contract is call AutoLine

The Techops streams are being `yank`ed. Not created.

```suggestion
summary: Setting PSM-GUSD-A Debt Ceiling to zero and removing from AutoLine, Coinbase Custody (RWA014-A) Debt Ceiling increase, WBTC-A, WBTC-B, and WBTC-C Stability Fee changes, SBE parameter changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Cancel Vesting Streams for Steakhouse and TechOps, Set up Ecosystem Facilitator Vesting Streams and Spark proxy-spell
```

date: 2023-11-29T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] PSM-GUSD-A removal from Cash Stablecoins, Coinbase Custody Debt Ceiling increase, various parameter changes, RWA Foundation Dao Resolution, Andromeda legal expenses, and Spark proxy-spell - November 29, 2023

Reviewer: to match line 1 suggestion

Changed to title case

```suggestion
# [Executive Proposal] PSM-GUSD-A Debt Ceiling Reduction, Coinbase Custody Debt Ceiling Increase, Various Parameter Changes, RWA Foundation Dao Resolution, Andromeda Legal Expenses, Vesting Stream Updates and Spark Proxy-Spell - November 29, 2023
```

The Facilitators, Sidestream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary
	
If this executive proposal passes, the following actions will occur within the Maker Protocol:
- PSM-GUSD-A will be removed from the Cash Stablecoin list by setting the [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) to zero and removing the vault from the [Debt Ceiling Instant Access (autoline)](https://manual.makerdao.com/module-index/module-dciam).

> - Coinbase Custody (RWA014-A) [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased to 1.5 billion.

Reviewer: suggest adding units

```suggestion
- Coinbase Custody (RWA014-A) [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased to 1.5 billion DAI.
```

- WBTC-A, WBTC-B, and WBTC-C [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%.
- The [hop (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) and [bump (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) SBE parameters will be changed to optimize for cost efficiency.
- A RWA Foundation Dao Resolution will be approved.

> - A transfer will be made to cover Andromeda Legal expenses.

Reviewer: nit: "Legal Expenses" or "legal expenses", not "Legal expenses"

> - A transfer will be made to cover Andromeda Legal expenses.

Reviewer: **Not blocking**

Consider adding the value since other entries in the Executive Summary have values specified.

> - MKR and DAI Vest Streams for Steakhouse and TechOps will be Ceased.

Reviewer: nit: cancel or yank would be preferable terms to cease. It also does not need to be capitalized.

- Ecosystem Facilitator Vest Streams will be set up.

> - A Spark Protocol proxy-spell will be excecuted to address a identified discrepancy in the DAI Market.

Reviewer: spelling and grammar changes

```suggestion
- A Spark Protocol proxy-spell will be executed to address an identified discrepancy in the DAI Market.
```

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

> ### Remove PSM-GUSD-A from the Cash Stablecoin list

Reviewer: see non blocking comment above re: wording

*Authorization*: [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2)\
*Proposal*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882#reduce-psm-gusd-a-debt-ceiling-8) "Stability Scope Parameter Changes"

If this executive proposal passes, in line with the recent amendments in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#7-2-1-3-1a), PSM-GUSD-A will be removed from the Cash Stablecoin list by performing the following actions:

> - The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to *zero*.

Reviewer: **Non Blocker**

nit: we prefer numerical "0" since it matches the code. https://github.com/makerdao/spells-mainnet/pull/372/files#diff-c9c3998420c475f01de841ca5283ea49ce6c99080a8c6a3fdae77b097bef6778R126-R127

```suggestion
- The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be set to *0*.
```

> - The vault will be removed from the [Debt Ceiling Instant Access (autoline)](https://manual.makerdao.com/module-index/module-dciam).

Reviewer: **Non blocker**

nit: in reality, it is the collateral type that is removed from the AutoLine. Although in the case of PSMs these are one in the same. This would not be correct if this were an ETH-based collateral.

### Increase Coinbase Custody (RWA014-A) Debt Ceiling

*Authorization*: [Ecosystem Approval](http://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882/2)\
*Proposal*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-7/22882#increase-rwa014-a-coinbase-custody-debt-ceiling-9) "Stability Scope Parameter Changes"

If this executive proposal passes, in line with the recent amendments in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#7-2-1-3-1a), the following parameters for Coinbase Custody (RWA014-A) will be changed:

> - The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased by 1 billion DAI from 500 million to *1.5 billion DAI*.

Reviewer: add missing unit

```suggestion
- The [Debt Ceiling (line)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) will be increased by 1 billion DAI from 500 million DAI to *1.5 billion DAI*.
```

- The WBTC-B [Stability Fee (SF)](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%, from 6.36% to *6.29%*.
- The WBTC-C [Stability Fee (SF)](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) will be decreased by 0.07%, from 5.61% to *5.54%*.

### Smart Burn Engine Parameter Optimization

*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-3/22876/2)\
*Proposal*: [Forum post](https://forum.makerdao.com/t/smart-burn-engine-transaction-analysis-parameter-reconfiguration-update-3/22876) "Smart Burn Engine - Transaction Analysis & Parameter Reconfiguration Update #3"

If this executive proposal passes, following the recommendation of the Stability Scope Advisory Council Member, BA Labs, the following SBE parameters will be changed:

- The [bump (lot size)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-3) parameter will be increased by 10,000, from 20,000 to *30,000*.

> - The [hop (frequency)](https://mips.makerdao.com/mips/details/MIP104#9-1-3-1) parameter will be increased by 9,460, from 6,308 to *15,768*.

Reviewer: Add units. `bump` is measured in DAI. `hop` is measured in seconds.

*Proposal*: [Forum post](https://forum.makerdao.com/t/dao-resolution-rwa-foundation-service-provider-changes/22866) "DAO Resolution - RWA Foundation Service Provider Changes"

If this executive proposal passes, the following RWA Foundation Dao Resolution to change the service provider will be approved:

- The [DAO Resolution](https://gateway.pinata.cloud/ipfs/QmPiEHtt8rkVtSibBXMrhEzHUmSriXWz4AL2bjscq8dUvU) with IPFS hash `QmPiEHtt8rkVtSibBXMrhEzHUmSriXWz4AL2bjscq8dUvU` will be approved.

### DAI transfer to cover Andromeda Legal expenses

*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577/4)\
*Proposal*: [Forum post](https://forum.makerdao.com/t/project-andromeda-legal-expenses-ii/22577) "Project Andromeda - Legal Expenses II"

If this executive proposal passes, the following transfer will be performed to cover Andromeda (RWA015-A) Legal expenses:

- *201,738 DAI* will be transfered to [0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767](https://etherscan.io/address/0xc4dB894A11B1eACE4CDb794d0753A3cB7A633767).

### Cease MKR and DAI Vest Streams for Steakhouse and TechOps

> *Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\

Reviewer: ```suggestion
*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)
```

The backslash is being rendered in the rich diff, this is presumably not desirable

> If this executive proposal passes, in line with the recent amendments in the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a), the following DSS Vest Streams will be halted by calling the Yank function:

Reviewer: ```suggestion
If this executive proposal passes, in line with the recent amendments in the [Governance Scope](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a), the following DSS Vest Streams will be halted by calling the yank function:
```

function names are lower case in Solidity

- Steakhouse *DAI* stream *ID 19*
- TechOps *DAI* stream *ID 18*
- Steakhouse *MKR* stream *ID 32*
- TechOps *MKR* stream *ID 33*

### Set up MKR and DAI Vest Streams for Ecosystem Facilitator

> *Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)\

Reviewer: ```suggestion
*Authorization*: [MIP 113](https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a)
```

The backslash is being rendered in the rich diff, this is presumably not desirable

|-----------|---------------------------------------------|------------|-------|-----------------------------------------------------------------------------------------------------------------------|
| Ecosystem | 2023-12-01 00:00:00 to 2024-11-30 23:59:59 | 504,000.00 | DAI   | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) |
| Ecosystem | 2023-12-01 00:00:00 to 2024-11-30 23:59:59 |     216.00 | MKR   | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) |

### Spark Protocol Proxy-Spell

*Authorization*: [Ecosystem Approval](https://forum.makerdao.com/t/accounting-discrepancy-in-the-dai-market/22845/2)\
*Proposal*: [Forum post](https://forum.makerdao.com/t/accounting-discrepancy-in-the-dai-market/22845) "Accounting Discrepancy in the DAI Market"

If this executive proposal passes, a Spark Protocol proxy-spell with address [0x68a075249fA77173b8d1B92750c9920423997e2B](https://etherscan.io/address/0x68a075249fA77173b8d1B92750c9920423997e2B) will be excecuted to address a identified discrepancy in the DAI Market.

> There is no impact of this issue on users and its severity is classified as low.

Reviewer: ```suggestion
If this executive proposal passes, a Spark Protocol proxy-spell with address [0x68a075249fA77173b8d1B92750c9920423997e2B](https://etherscan.io/address/0x68a075249fA77173b8d1B92750c9920423997e2B) will be excecuted to address a identified discrepancy in the DAI Market.

There is no impact of this issue on users and its severity is classified as low.
```

If a line break is intended here then you need an empty line in markdown, or to use a double space "  " at the end of the first line.
